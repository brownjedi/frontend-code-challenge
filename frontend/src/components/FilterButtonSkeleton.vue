<template>
  <div :class="styles.button">
    <skeleton />
  </div>
</template>

<script lang="ts">
import { defineComponent, useCssModule } from 'vue'
import Skeleton from './Skeleton.vue'

export default defineComponent({
  components: { Skeleton },
  setup() {
    const styles = useCssModule()
    return { styles }
  },
})
</script>

<style lang="scss" module>
.button {
  border: none;
  height: 2.375rem;
}
</style>
