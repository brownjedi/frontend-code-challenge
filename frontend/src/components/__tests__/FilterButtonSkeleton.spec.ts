import { shallowMount } from '@vue/test-utils'
import FilterButtonSkeleton from '@/components/FilterButtonSkeleton.vue'

describe('components/FilterButtonSkeleton.vue', () => {
  it('has the expected html structure', () => {
    const wrapper = shallowMount(FilterButtonSkeleton)
    expect(wrapper.element).toMatchSnapshot()
  })
})
