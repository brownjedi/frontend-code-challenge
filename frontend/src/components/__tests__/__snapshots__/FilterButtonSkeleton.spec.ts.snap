// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/FilterButtonSkeleton.vue has the expected html structure 1`] = `
<div
  class="button"
>
  <skeleton-stub />
</div>
`;
